### PR TITLE
Feat: Cart page is added

### PR DESCRIPTION
**Pull request by sakleprajwal**
Cart page is added to the E-commerce website.
Preview:

https://user-images.githubusercontent.com/40752468/154722807-fb129f46-8815-45ce-9265-73e619da6d06.mp4

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -4,6 +4,7 @@
 @import url(css/util.css);
```
Comment (krishanu2911) [inline style.css:4]: just a suggestion of add the util CSS at the last or else it will be overridden by the below CSS files

**Review by sadathshariff — APPROVED** [review]
Everything looks good

**Review by krishanu2911 — APPROVED** [review]
Nice design and clean code written good class naming, no changes required 
just added a small suggestion it might be helpful.

File: css/cart-page.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    font-family: var(--secondary-font);
+}
+
+.cart-price-breakdown{
+    margin-bottom: 1.5rem;
+    font-weight: 400;
+}
+
+.cart-price-breakdown h4{
+    font-weight: 400;
```
Comment (kotalwar-krishna) [inline css/cart-page.css:47]: for font weight you can use var() because same repeating

**Review by kotalwar-krishna — APPROVED** [review]
one saggestion be cosistent with unit. all other things are Good. Good to merge

**Pull request merged**